src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/Subscriptions/DependencySubscriptionsHost.cs

```diff
@@ -274,11 +273,7 @@ private void OnDependenciesSnapshotChanged(IDependenciesSnapshot snapshot)
             DependenciesSnapshot newSnapshot;
```

Comment:
Yes I'm aware of this - you'll see I did not introduce this method in this PR. I just kept the changes small for review purposes.